src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -5,7 +5,7 @@
 "123.400"
 
 >> Text(Blank(), "#.000")
-""
+Blank()
```

Comment:
In the latest discussions with PMs we agreed that if there is only one argument to the Text function (i.e., it's being used as a conversion function) then blank values are propagated. However, if we explicitly ask for a format, as in this example, then the result should be an empty string (`""`)

Reply:
Updated to handle this. It was not clear in the problem. Thanks for explaining here.